pages/5g-core-network-components/tutorials/5gnetwork.md

```diff
@@ -10,17 +10,21 @@ nav_order: 0
 # Tutorial - 5G Network Setup with COTS UE
 
 ## Introduction
-These are the generic instructions to setup a 5G network using Open5GS and srsRAN. An Ettus X310 USRP and a Pixel 8 phone are used. 
+
+These are the generic instructions to setup a 5G network using Open5GS and srsRAN. An Ettus X310 USRP and a Pixel 8
```

Comment:
Should be "to set up" ("setup" is only one word when used as a noun).

Reply:
Thanks, fixed.